unison-src/new-runtime-transcripts/codeops.md

```diff
@@ -0,0 +1,53 @@
+
+Test for code serialization operations.
+
+```ucm:hide
+.> builtins.merge
+.> cd builtin
+```
+
+Define a function, serialize it, then deserialize it back to an actual
```

Comment:
How about some more tests for this transcript (that round trip is `Universal.==` for a variety of definitions), following https://github.com/unisonweb/unison/blob/trunk/unison-src/new-runtime-transcripts/builtins.output.md for inspiration. 

Basically, you create a bunch of `test>` watch expressions, add them all to the codebase, then do one `.> test` at the end.

Reply:
I've added some more complexity to the thing that's serialized. I can't run deserialization in a test block, though, because it involves IO, no?

Reply:
That's right. 

@stew has an implementation of a `io.test` command that lets you run I/O-based tests, so let's leave this until that merges. I'll create an issue to track.